### PR TITLE
Contact form fixes

### PR DESCRIPTION
**Pull request by braamgenis**
Resolves issues #34 and #35 as well as an issue I picked up with having multiple recaptchas on a page.

### CODE REVIEW

File: widgets/contact/js/contact.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+var SiteOriginContactForm = {
```
Comment (gregpriday) [inline widgets/contact/js/contact.js]: @braamgenis For the most part, we've been doing indenting using 4 spaces in our Javascript files. This file seems to be using 2 space indenting. Although we really should be using the WordPress [JS best practices](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/) and using tab indents.

At some point we should do a more thorough run through all our code to make sure we're following these guidelines.

File: widgets/contact/tpl/default.php [hunk 2]
```diff
@@ -35,25 +42,13 @@
...
-			</script>
-			<?php endif; ?>
+			<div class="sow-recaptcha"
+				 data-config="<?php echo esc_attr( json_encode( $settings ) ) ?>"></div>
 		<?php endif; ?>
 
 		<div class="sow-submit-wrapper <?php if( $instance['design']['submit']['styled'] ) echo 'sow-submit-styled' ?>">
 			<input type="submit" value="<?php echo esc_attr( $instance['settings']['submit_text'] ) ?>"
-				   class="sow-submit <?php echo 'js-sow-submit-' . $instance['_sow_form_id'] ?>" <?php if( $use_recaptcha && !is_admin() ) echo 'disabled="true"'; ?>>
+				   class="sow-submit" <?php if( $use_recaptcha && !is_admin() ) echo 'disabled="true"'; ?>>
```
Comment (gregpriday) [inline widgets/contact/tpl/default.php]: Are we making considerations for browsers with Javascript disabled? Wont this button always be disabled for these users?

Come to think of it, how does Recaptcha handle non JS browsers? Does it handle them at all? Or has the world decided disabling JS just isn't cool?

File: widgets/contact/js/contact.js (hunk 1 above)
Comment (braamgenis) [inline widgets/contact/js/contact.js]: Ah, yeah, I'm going to add an .editorconfig file. Should help prevent this in future.

File: widgets/contact/tpl/default.php (hunk 2 above)
Comment (braamgenis) [inline widgets/contact/tpl/default.php]: Nope. I don't think many sites will work with JS disabled these days. Besides the fact that the Recaptcha API is loaded as a JS file. :grin:

Comment (gregpriday) [inline widgets/contact/tpl/default.php]: Something that might come up is a user wanting to disable Recaptcha so they can offer their non JS users access. Ideally we want to offer users a non JS mode of operation, even if it becomes slightly limited.

Worth reading the arguments here to decide how far we want to go with JS support - http://www.punkchip.com/why-support-javascript-disabled/

**Pull request merged**